jcl/src/java.base/share/classes/openj9/management/internal/ThreadInfoBase.java

```diff
@@ -0,0 +1,398 @@
+			boolean inNativeVal, long blockedCountVal, long blockedTimeVal, long waitedCountVal, long waitedTimeVal,
+			String lockNameVal, long lockOwnerIdVal, String lockOwnerNameVal, StackTraceElement[] stackTraceVal,
+			LockInfoBase lockInfoVal, MonitorInfoBase[] lockedMonitorsVal, LockInfoBase[] lockedSynchronizersVal
+/*[IF Sidecar19-SE]*/
+			, boolean daemonVal, int priorityVal
+/*[ENDIF]*/
+	) {
+		threadId = threadIdVal;
+		nativeTId = -1;
+		threadName = threadNameVal;
+		threadState = threadStateVal;
+		suspended = suspendedVal;
+		inNative = inNativeVal;
+		blockedCount = blockedCountVal;
+		blockedTime = blockedTimeVal;
+		waitedCount = waitedCountVal;
+		waitedTime = waitedTimeVal;
+		lockName = lockNameVal;
+		lockOwnerId = lockOwnerIdVal;
+		lockOwnerName = lockOwnerNameVal;
+		stackTrace = stackTraceVal;
+		blockingLockInfo = lockInfoVal;
+		lockedMonitors = lockedMonitorsVal;
+		lockedSynchronizers = lockedSynchronizersVal;
+/*[IF Sidecar19-SE]*/
+		daemon = daemonVal;
+		priority = priorityVal;
+/*[ENDIF]*/
+	}
+
+	@SuppressWarnings("unused")
+	/* used by native code */
+	private ThreadInfoBase(Thread thread, long nativeId, int state, boolean isSuspended, boolean isInNative,
+			long blockedCountVal, long blockedTimeVal, long waitedCountVal, long waitedTimeVal, StackTraceElement[] stackTraceVal,
+			Object blockingObject, Thread blockingObjectOwner, MonitorInfoBase[] lockedMonitorsVal,
+			LockInfoBase[] lockedSynchronizersVal) {
+		threadId = thread.getId();
+		nativeTId = nativeId;
+		threadName = thread.getName();
+		/*[IF Sidecar19-SE]*/
+		daemon = thread.isDaemon();
+		priority = thread.getPriority();
+		/*[ENDIF]*/
+
+		// Use the supplied state int to index into the array returned
+		// by values() which should be all thread states in the order
+		// they have been declared. Doing this rather than just issuing a
+		// call to thread.getState() because the Thread state may have
+		// changed after the VM issued the call to this method.
+		threadState = Thread.State.values()[state];
+		suspended = isSuspended;
+		inNative = isInNative;
+		blockedCount = blockedCountVal;
+		blockedTime = blockedTimeVal;
+		waitedCount = waitedCountVal;
+		waitedTime = waitedTimeVal;
+
+		if (blockingObjectOwner != null) {
+			lockOwnerId = blockingObjectOwner.getId();
+			lockOwnerName = blockingObjectOwner.getName();
+		} else {
+			lockOwnerId = -1;
+			lockOwnerName = null;
+		}
+
+		stackTrace = stackTraceVal;
+		String lockNameTemp = null;
+		if (blockingObject != null) {
+			lockNameTemp = blockingObject.getClass().getName() + '@'
+					+ Integer.toHexString(System.identityHashCode(blockingObject));
+		}
+
+		lockedMonitors = lockedMonitorsVal;
+		lockedSynchronizers = lockedSynchronizersVal;
+
+		if (blockingObject != null) {
+			blockingLockInfo = new LockInfoBase(blockingObject.getClass().getName(),
+					System.identityHashCode(blockingObject));
+		} else {
+			blockingLockInfo = null;
+		}
+		if (null != blockingLockInfo) {
```

Comment:
Please be consistent: in the method all other null tests have null on the right.